features/bsm/service/impl/src/main/java/org/opennms/netmgt/bsm/service/internal/BusinessServiceManagerImpl.java

```diff
@@ -515,6 +581,15 @@ private OnmsMonitoredService getMonitoredServiceEntity(Integer serviceId) throws
         return monitoredService;
     }
 
+    private OnmsApplication getApplicationEntity(Integer applicationId) throws NoSuchElementException {
```

Comment:
This should return the entity shouldn't it?

Reply:
It does. What do you mean?

Reply:
Nevermind.